setup.py

```diff
@@ -36,6 +36,10 @@
     "viewer": [
         'windows-curses;platform_system=="Windows" and platform_python_implementation=="CPython"'
     ],
+    "mf4": [
+        'asammdf>=6.0.0;platform_python_implementation=="CPython" and python_version<"3.11"',
+        'numpy;platform_python_implementation=="CPython" and python_version<"3.11"',
```

Comment:
`asammdf` will declare the version of numpy it needs. Not sure about the platform and python version requirement either?

Reply:
I think i added this so CI does not fail on PyPy and CPython 3.11. But there's probably a better solution for that. I admit this is ugly.